### DISCUSSION TITLE
Sysmon Alerts created from playbook doesn´t have correct timestamp

### DISCUSSION THREAD

**Discussion by jot49** [Security-Onion-Solutions/securityonion#2836] [Unsupported Versions]
Hi all,

I import Sysmon logs using Winlogbeat and use various playbooks to generate alerts on that data.
But the alerts only have a timestamp of 0001-01-01..
I have to open every single alert to see when the alert was generate. 
Has anyone an idea what the reason could be?

![image](https://user-images.githubusercontent.com/73352811/106738836-b9f4ab80-6618-11eb-9881-eedaccc6d9c0.png)

**Comment by 0nnyx** [comment]
Already reported https://github.com/Security-Onion-Solutions/securityonion/discussions/2069 and fix in progress
